### DISCUSSION TITLE
"Stripplot" using dodge?

### DISCUSSION THREAD

**Discussion by knutwannheden** [observablehq/plot#1074] [Q&A] [answered]
Hi,

I have some past experience with Altair (Vega-Lite) where I have occasionally used [stripplot](https://altair-viz.github.io/gallery/stripplot.html) (see screenshot below).

<img width="431" alt="image" src="https://user-images.githubusercontent.com/408765/194726543-9281812f-c676-41bf-8b01-2f8b84267f38.png">

Looking at the documentation I found [dodge](https://observablehq.com/@observablehq/plot-dodge?collection=@observablehq/plot), which looks very nice. So I was trying to be able to use a continuous value for the `y` value and then use dodge for the `x` value, while at the same time I also want to split the data into separate beeswarms like in the screenshot.

I got this working using facets, but I thought that there might be a better way to do this. I tried using the `z` channel, but that doesn't seem to be available here.

**Comment by mbostock** [comment, marked as answer]
I think facet is your only option here; that’s the only way to group the points before dodging. The dodge transform doesn’t (currently) provide an auxiliary way to group data. Here’s my take:

https://observablehq.com/@observablehq/stripplot

Also, this helped me find a bug [#1075](https://github.com/observablehq/plot/pull/1075). Thanks for sharing your question!

**Reply by knutwannheden** [reply]
Thanks for your example!
